.github/workflows/run-integ-tests.yaml

```diff
@@ -188,7 +240,7 @@ jobs:
           PULUMI_CONFIG_PASSPHRASE: ""
       - name: pulumi up (upgrade path)
         if: matrix.mode == 'upgrade'
-        uses: klothoplatform/gh-action-retry@v1
+        uses: CloudCompilers/gh-action-retry@v1
```

Comment:
Does the new org name not work with this?

Reply:
Whoops, yes it would/should. I missed this, thanks!